<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="TabsContainer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>9</y>
      <width>650</width>
      <height>391</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>650</width>
      <height>400</height>
     </size>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="myDocsTab">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Meus Documentos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>70</y>
        <width>201</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Assinaturas:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="signList"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>271</width>
        <height>331</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="docsList"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="addDocTab">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>419</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QPushButton" name="addUserButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Adicionar assinante</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="userNameInput">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>419</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Nome do documento:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="docNameInput">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="addDocButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>260</y>
        <width>419</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Adicionar Documento</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>419</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QPushButton" name="selectDocButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Selecionar Arquivo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="docNameLabel">
         <property name="maximumSize">
          <size>
           <width>155</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Nenhum arquivo selecionado</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="signDocTab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>241</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Documentos para assinar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="docsToSignList"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOp_es">
    <property name="title">
     <string>Opções</string>
    </property>
    <addaction name="actionSignout"/>
   </widget>
   <widget class="QMenu" name="menuDados">
    <property name="title">
     <string>Banco de Dados</string>
    </property>
    <addaction name="actionUsuarios"/>
    <addaction name="separator"/>
    <addaction name="actionDocumentos"/>
    <addaction name="separator"/>
    <addaction name="actionAssinaturas"/>
   </widget>
   <addaction name="menuOp_es"/>
   <addaction name="menuDados"/>
  </widget>
  <action name="actionUsuarios">
   <property name="text">
    <string>Usuarios</string>
   </property>
  </action>
  <action name="actionDocumentos">
   <property name="text">
    <string>Documentos</string>
   </property>
  </action>
  <action name="actionAssinaturas">
   <property name="text">
    <string>Assinaturas</string>
   </property>
  </action>
  <action name="actionSignout">
   <property name="text">
    <string>Encerrar sessão</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <signal>closing()</signal>
 </slots>
</ui>
